### PR TITLE
docs: :memo: post on how to write learning shorts

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

- This PR adds a post on writing learning shorts. Learning shorts are some ways we can share knowledge internally and externally and something we should try to do whenever we learn something new (e.g. during our learning breaks).

### CODE REVIEW

**Review by signekb — COMMENTED** [review]
Ah, nice! This definitely makes it more clear to me what a learning short is 😅  
I have some suggestions and questions as always.

File: guides/learning-shorts.qmd [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+the aim is also to spread knowledge and teach others on concepts within
+research data engineering, research software engineering, analytic
+programming/coding, data management, and open research practices (like
+FAIR data).
+
+One tool we use as part of our teaching strategy is called
+micro-learning or micro-lessons (which we call "learning shorts" or
+"shorts"). Learning shorts are short, focused lessons that are designed
+to teach a single concept or skill, with minimal or at least explicitly
+described outside knowledge.
```
Comment (lwjohnst86) [inline guides/learning-shorts.qmd:21]:
```suggestion
described prerequisite knowledge.
```

File: guides/learning-shorts.qmd [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+larger learning goal by encouraging behaviour and culture change over a
+persistent, longer period of time.
+
+## Learning shorts template
+
+Within the `shorts/` folder of the learning section contains a template
+for writing these shorts called `_template.qmd`. The template is a
+Markdown file that contains the following structure:
+
+``` markdown
```
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply guides/learning-shorts.qmd:51]: Good point!

File: guides/learning-shorts.qmd [hunk 3]
```diff
@@ -45,56 +45,26 @@ persistent, longer period of time.
...
+    gain. It should be as simple and short as needed to convey the
+    learning goal. It should be written in a form based on [Bloom's
+    Taxonomy](https://bloomstaxonomy.net/) (or a similar structure).
 -   **Lesson content**: This is the main part of the short, where the
     core teaching content is written. It should be broken down into
-    sections (`###`). Try to make use of images, diagrams, tables, and
-    other visual aids to help explain the content. If it is code based,
-    make sure to actually show code.
+    sections (with several headers denoted by `###`). Try to make use of
+    images, diagrams, tables, and other visual aids to help explain the
```
Comment (signekb) [inline guides/learning-shorts.qmd:64]: Oh, I wonder what "other visual aids" could be? Videos?

File: guides/learning-shorts.qmd [hunk 4]
```diff
@@ -45,56 +45,26 @@ persistent, longer period of time.
...
+    content. If it is code based, make sure to actually show code.
```
Comment (signekb) [inline guides/learning-shorts.qmd:65]:
```suggestion
    content. If it is code based, make sure to show the actual code.
```

**Review by signekb — COMMENTED** [review]
Very nice! Just two minor things :))

**Review by K-Beicher — DISMISSED** [review]
Shorts now make a lot more sense to me :-)

File: guides/learning-shorts.qmd [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+-   **Learning goal**: This is the most important part of the short,
+    aimed to inform the reader of what knowledge or skill will they
+    gain. It should be as simple and short as needed to convey the
+    learning goal. It should be written in a form based on [Bloom's
+    Taxonomy](https://bloomstaxonomy.net/) (or a similar structure).
+-   **Lesson content**: This is the main part of the short, where the
+    core teaching content is written. It should be broken down into
+    sections (with several headers denoted by `###`). Try to make use of
+    images, diagrams, tables, and other visual aids to help explain the
```
Comment (lwjohnst86) [inline guides/learning-shorts.qmd:64]:
```suggestion
    sections (with several headers denoted by `###`). Try to make use of
    visual aids like images, diagrams, and tables to help explain the
```

**Pull request merged**